osc_sdk_python/authentication.py

```diff
@@ -2,13 +2,15 @@
 import hashlib
 import hmac
 
+DEFAULT_USER_AGENT = "osc-sdk-pythons/0.9.16-git"
```

Comment:
- typo (`pythonS`)
- please extract version to dedicated string or dedicated file (better)
- why "git" ?

Reply:
Because I had to made a decision: I could either have used last release number, but it would have been a lie, because we have commit on top of it.
I could have used 0.9.16 but it would also be a lit because it wasn't release yet, so my idea was to use next version + -git to explicit that this is not the release version.